superset-frontend/package.json

```diff
@@ -209,10 +207,13 @@
     "@emotion/babel-preset-css-prop": "^10.0.27",
     "@hot-loader/react-dom": "^16.13.0",
     "@istanbuljs/nyc-config-typescript": "^1.0.1",
+    "@storybook/addon-actions": "^6.0.13",
```

Comment:
qq: should these be devDependencies? I assume webpack is removing them from the prod packages with tree-shaking, but it might be good to specify these as required for dev only

Reply:
They are dev dependencies already. Maybe Github's code folding got ya? :D